docs/guides/linux_thread_connectivity.md

```diff
@@ -0,0 +1,147 @@
+# Configuring Thread network on a Linux machine
+
+In order to use a CHIP controller such as
+[Python CHIP Controller](../../src/controller/python/README.md) to control
+CHIP-enabled Thread devices from a Linux host, the host must be equipped with a
+Thread radio device, a new Thread network must be created and packets addressed
+to devices within the network must be routed via the radio device. The article
+describes how to achieve that on a Linux machine with Ubuntu 20.04, using
+[nRF52840 Dongle](https://www.nordicsemi.com/Software-and-tools/Development-Kits/nRF52840-Dongle)
+as the radio device.
+
+<hr>
+
+-   [Overview](#overview)
+-   [Configuring OpenThread Radio Co-Processor device](#configuring-openthread-radio-co-processor)
+-   [Configuring OpenThread Border Router](#configuring-openthread-border-router)
+-   [Testing](#testing)
+
+<hr>
+
+## Overview
+
+The goal of the article is to prepare a setup presented in the diagram below:
+
+![linux_thread_connectivity](../images/linux_thread_connectivity.svg)
+
+Connectivity between Python CHIP Controller and Thread devices is assured in the
+following way:
+
+-   CHIP messages generated by the CHIP controller are routed via a virtual
+    network interface **otbr0** to
+    [OpenThread Border Router container](https://openthread.io/guides/border-router/docker)
+    in Docker.
+-   OpenThread Border Router container uses nRF52840 Dongle with
+    [OpenThread Radio Co-Processor](https://openthread.io/platforms/co-processor)
+    firmware to transmit CHIP messages to the Thread network.
+-   CHIP-enabled Thread devices connected to the same Thread network receive and
+    consume CHIP messages.
+
+The goal will be achieved by completing the following two steps:
+
+1.  Programming OpenThread Radio Co-Processor (RCP) firmware onto an nRF52840
+    Dongle.
+2.  Configuring OpenThread Border Router container in Docker.
+
+Note that if you're using a Raspberry Pi as your testing platform, you can
+execute the first step (building and programming the RCP firmware) on a more
+powerful Linux machine to shorten the firmware build time.
+
+## Configuring OpenThread Radio Co-Processor
```

Comment:
I would move this section into a separate doc title something like "nordic OpenThread RCP". That way each manufacturer can add its own RCP doc. Otherwise this document would become way too big if we add the building step for every possible OT RCP.

Reply:
In general, this is a good idea and I agree. However, I would propose to proceed with the current form to address CSG need, and if someone would like to add instruction for another device, we could create the necessary abstraction then.

Reply:
@LuDuda Copy-pasting the content into another file is a matter of seconds. And as of the "if someone would like to add instructions for another device", obviously we want to add instructions for our own rcp. My point is since CHIP is a common effort from the IoT industry, general documentation shouldn't be manufacturer specific. Having general documentation promoting the use of a single manufacturer without any prior consensus should never happen.

Reply:
My point was only to create abstraction when we have examples for at least two platforms,
rather than now when we don't. But anyway, let us address this now.

Reply:
@jepenven-silabs I think your judgement is a bit unfair. I'm simply not familiar with solutions provided by other vendors, so I can't provide an objective overview of available options and will unwittingly "promote" Nordic's solutions. Every manufacturer can, however, contribute by amending the documentation we provide and/or contact us to agree on a common shape of it. It seems that most CHIP developers prefer testing CHIP on Wi-Fi devices for obvious reasons, so any effort towards helping community with testing Thread devices is valuable for all Thread device manufacturers. Thus we're open to any cooperation in that regard :). 

Btw, I extracted the section you mentioned to a separate document.

Reply:
@Damian-Nordic I understand completely your point. However I simply wanted us to be ahead of the problem. Having a "generic" *.md for the Border router that refer to a set of manufacturer specific *.md for each possible rcp setup seemed like the easiest way to prevent having a huge border router doc with all of the possible rcp config directly inside of it. Also by dividing it into multiples files we are sending the message to other manufacturer : "If you want to add your RCP config add a *.md with your own setup"